parser/parser.y

```diff
@@ -10705,6 +10705,20 @@ AdminStmt:
 			JobIDs: $5.([]int64),
 		}
 	}
+|	"ADMIN" "PAUSE" "DDL" "JOBS" NumList
```

Comment:
Please add tests like https://github.com/pingcap/tidb/blob/master/parser/parser_test.go#L827.

Reply:
Sure, on it.

Reply:
done